.gitignore

```diff
@@ -9,6 +9,7 @@
 Testing
 __
 tests/failed
+2
```

Comment:
This file '2' is due to a bug. It should not be here. So I would prefer we do not exclude it from the git.
Please at the moment, just avoid to push  it.